nano/nano_wallet/entry.cpp

```diff
@@ -358,13 +358,23 @@ int main (int argc, char * const * argv)
 					{
 						data_path = nano::working_path ();
 					}
+					size_t batch_size;
+					if (vm.count ("batch_size"))
+					{
+						batch_size = (vm["batch_size"].as<size_t> ());
+					}
+					else
+					{
+						batch_size = 512;
```

Comment:
I guess no need to set 512 as this is sideband_batch_size default value

Reply:
Probably all this can be shortened to something like
`if (vm.count ("batch_size"))
{
	flags.sideband_batch_size = (vm["batch_size"].as<size_t> ());
}`